_includes/header.html

```diff
@@ -6,11 +6,14 @@
   </button>
   <!-- Menu -->
   <div class="collapse navbar-collapse">
-      <a class="navbar-brand" href="/">HelioPython</a>
+      <a class="navbar-brand" href="/">PyHC</a>
       <ul class="navbar-nav ml-auto">
         <li class="nav-item">
           <a class="nav-link {% if page.url == '/' %}active{% endif %}" href="/">Home</a>
         </li>
+          <li class="nav-item">
```

Comment:
I don't like having the gallery link in the navbar as clicking it navigates to a an external page. Since the gallery is for showing off the capabilities of the different projects, I think it would be better to add it as a link at the top of the projects page.

Reply:
Depends what you mean by external page. It is under the same domain. I would rather have it more up and center than hidden under projects but I agree that it would be better if the gallery page looked more like the main page (maybe used the same header?) and allowed the user to go back to the main page.

Reply:
It is now on the project page.